docs/Hyprland.1.rst

```diff
@@ -0,0 +1,75 @@
+========
+Hyprland
+========
+
+---------------------------------
+Dynamic tiling Wayland compositor
+---------------------------------
+
+:Date: 15 Jul 2022
+:Copyright: Copyright (c) 2022, vaxerski
+:Version: 0.7.1beta
+:Manual section: 1
+:Manual group: HYPRLAND
+
+SYNOPSIS
+========
+
+``Hyprland`` [arg [...]].
+
+DESCRIPTION
+===========
+
+``Hyprland`` is a dynamic tiling Wayland compositor based on
+wlroots that doesn't sacrifice on its looks.
+
+NOTICE
+======
+
+Hyprland is still in pretty early development compared to some other Wayland compositors.
+
+Although Hyprland is pretty stable, it may have some bugs.
+
+CONFIGURATION
+=============
+
+For configuration information please see <`https://github.com/hyprwm/Hyprland/wiki`>.
+
+LAUNCHING
+=========
+
+You can launch Hyprland by either going into a TTY and executing ``Hyprland``, or with a login manager.
+
+`IMPORTANT`: Do `not` launch ``Hyprland`` with `root` permissions (don't `sudo`)
```

Comment:
I think this is not that important, as most users won't do that. And even if there are users who do, the program will tell them before they do anything stupid.

Reply:
For this one I removed all the extra stuff that doesn't really make sense in a man page. Since it was only one line I just rolle d the `You can launch` line up into the description block